src/App.tsx

```diff
@@ -158,6 +158,14 @@ const App = () => {
     }
   }, [fetchDataStatus, isBackedUp, isRestored])
 
+  useEffect(() => {
```

Comment:
This isn't related to this PR, but it's been driving me crazy.

Reply:
It seems to work fine, not sure if 5s is the right number though.

Reply:
Hmm yeah I think mine is usually like a second or two. I guess if we want to do fixed maybe 2 seconds?

Reply:
Oh or this is only to handle the case it takes longer than 5 seconds?

Reply:
Yeah, this is so that at worst it will show 5 seconds.